types/src/test/scala/aqua/types/TypeSpec.scala

```diff
@@ -152,13 +152,13 @@ class TypeSpec extends AnyFlatSpec with Matchers {
     val three: Type = ArrowType(ProductType(bool :: f32 :: Nil), ProductType(u64 :: Nil))
     val four: Type = ArrowType(ProductType(bool :: f32 :: Nil), ProductType(u32 :: Nil))
 
-    accepts(one, two) should be(true)
-    accepts(two, one) should be(false)
+    accepts(one, two) should be(false)
```

Comment:
Why these tests changed? I supposed all the arrow tests should remain intact. It might be due to changes in codomain variance (maybe they're not made properly).